### ISSUE TITLE
The reason for doubling the size for input hint during eval

### ISSUE THREAD

**Issue by wigwit** [Diouo/MotionLab#5]
Hi! Thank you for your amazing work!
I am wondering if there is any particular reason you choose to double the size of input_hint by concatenating zeros before the actual mask in the function `def eval_rectified_flow_hint(self,batch)` from`rfmotion.py`, from [here](https://github.com/Diouo/MotionLab/blob/bf753b26cee6647e7b6d9391cd1c2830e90f8fe9/rfmotion/models/modeltype/rfmotion.py#L764). In line 781-782, you initialized the list as this

```
input_hint = [torch.zeros((len(text), seq_len, 66), device=target_motion.device)] 
hint_lengths = [torch.zeros((len(text), seq_len), device=target_motion.device)]
```

and this holds the same in the function `def eval_rectified_flow_inbetween(self,batch)` in line 1246-1247 [(here).](https://github.com/Diouo/MotionLab/blob/bf753b26cee6647e7b6d9391cd1c2830e90f8fe9/rfmotion/models/modeltype/rfmotion.py#L1246)  And I noticed that this the list is intialized as empty during training, is there a particular reason why there's extra zero added during evaluation?

**Comment by Diouo** [comment]
Hi Qianxia, this is for Classifier-Free Guidance, you may check Section 5.4 in our paper or directly Google this trick.
